### DISCUSSION TITLE
How to type a colon (:) in a publication title?

### DISCUSSION THREAD

**Discussion by tangruize** [rendercv/rendercv#51] [Q&A] [answered]
Thanks for the great project!

I have a publication which contains a colon in the title. However, I encountered the following problem:

```txt
There is a YAML error in the input file!

None

None

mapping values are not allowed here

  in "<unicode string>", line 25, column 25:
          - title: XXXX: XXXX XXXX XXX XXX ... 
                            ^ (line: 25)
```

**Comment by sinaatalay** [comment, marked as answer]
Thank you! Could you try putting the title inside quotation marks in the YAML file? That should work because it seems the YAML parser doesn’t understand the colon is a part of the string.

**Comment by tangruize** [comment]
Thank you very much! It works now!
